### DISCUSSION TITLE
Failed to install refgenie

### DISCUSSION THREAD

**Discussion by ghost** [biostars/biostar-handbook-issues#188] [General]
Hi, I want to try to install the refgenie following the steps in the handbook.
However, it came out the errors:

error in refgenie setup command: use_2to3 is invalid.

I googled this error, it seems that the version after setuptools 58 does not support use_2to3. Maybe It needs to update setuptools to a version before 58...

Hmm I don't know is there anybody has such problem. Just a submission.

**Comment by ialbert** [comment]
this should probably be submitted to refgenie as an error, they will most likely fix it promptly

**Comment by ghost** [comment]
Thx for advice！

**Comment by ialbert** [comment]
https://github.com/refgenie/refgenie/issues/264
